src/containers/BooksForm.js

```diff
@@ -1,18 +1,56 @@
-const BooksForm = () => {
-  // eslint-disable-next-line no-unused-vars
-  const categoryCollection = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+import { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import randomIdGenerator from '../helpers/randomId';
+import { createBookAction } from '../redux/actions';
+import categoriesOptions from '../helpers/categoriesList';
+
+const BooksForm = ({ createBook }) => {
+  const [state, setState] = useState({ bookTitle: '', bookCategory: '' });
+  const handleChange = ({ target: { name, value } }) => {
+    setState({ ...state, [name]: value });
+  };
+  const handleSubmit = event => {
+    event.preventDefault();
```

Comment:
**[OPTIONAL]**
You can destructure, bookTille and BookCategory from the state to have a cleaner and non-repetitive code.
`const {bookTitle, bookCategory}=state;`

Reply:
Thnaks for the advice, Done :heavy_check_mark: